m2isar/backends/etiss/writer.py

```diff
@@ -55,7 +56,13 @@ def main():
 	for core_name, core in models.items():
 		logger.info("processing model %s", core_name)
 		output_path = output_base_path / spec_name / core_name
-		output_path.mkdir(exist_ok=True, parents=True)
+		try:
+			output_path.mkdir(parents=True)
+		except FileExistsError:
+			logger.warning("deleting existing ETISS architecture %s files and creating new ones", core_name)
```

Comment:
I don't think outputting a warning here is necessary, the files would be overwritten anyways.

Reply:
I am also actively deleting files that won't be replaced. But I can take that out. Should I delete it or change it to `.info(...)`?

Reply:
I just deleted it, see b6dcf29